sourcemash/frontend/static/js/views/item_card_view.js

```diff
@@ -70,16 +93,20 @@ Sourcemash.Views.ItemCardView = Backbone.View.extend({
             this.model.save({saved: false},
                 {success: this.savedToast});
         } else {
-            this.model.save({saved: true},
-                {success: this.savedToast});
-
-            mixpanel.track("Saved", { "Item Title": this.model.get('title'),
-                                    "Feed Title": this.model.feed.get('title') })
-
-            if (!this.model.feed.get('subscribed')) {
-                this.showSubscribeModal({'source':'saved'});
-            }
-        }
+            if (!this.user.get('id')) {
```

Comment:
In index.html, we have divs for top-nav, side-nav, and main. We'd add another div for register-modal. The registerModal view would be rendered in the initialize function of app_router, just like sidenav is at the moment.

Then, in default.js, we already have a function that is called on ajaxError. Right now it just renders error messages, but we'll add another part that opens the modal if the error code is a 401 Unauthorized Error. If yes, you just call $(".register-modal").openModal() or whatever it is, and send mixpanel an event with the url as the source.